docs/symbols.md

```diff
@@ -74,9 +73,10 @@ Ion text into binary.
 At any point during processing, there is a *current symbol table*, which
 is either a local symbol table or a system symbol table. At the start of
 input, the current symbol table is initialized to be the system symbol
-table for Ion 1.0. The current symbol table is only changed in two
-circumstances: (1) encountering a system identifier at the top-level, or
-(2) encountering a local symbol table at the top-level.
+table for the Ion version denoted by the Ion version marker. The current
+symbol table is only changed in two circumstances: (1) encountering a
+system identifier at the top-level, or (2) encountering a local symbol table
```

Comment:
I find the phrase "system identifier" (which was in the original text) to be rather ambiguous. I'd suggest swapping it out for "Ion version marker" if that's what it means.

Reply:
The term also shows up at L334:

> When immediately following an explicit system ID, a top-level struct whose first annotation is `$ion_symbol_table` is interpreted as a *local symbol table*.

I think that when-clause is incorrect, and that such a top-level struct is always an LST, even if it's not immediately following an IVM.

Reply:
@toddjonker Agreed, that should be fixed.